$NetBSD: patch-ae,v 1.2 2013/08/14 05:31:04 obache Exp $

* gtk_status_icon_set_tooltip is deprecated
* for libnotify-0.7 API change

--- src/gpk-watch.c.orig	2008-11-24 16:34:22.000000000 +0000
+++ src/gpk-watch.c
@@ -109,7 +109,7 @@ gpk_watch_refresh_tooltip (GpkWatch *wat
 	length = pk_task_list_get_size (watch->priv->tlist);
 	egg_debug ("refresh tooltip %i", length);
 	if (length == 0) {
-		gtk_status_icon_set_tooltip (GTK_STATUS_ICON (watch->priv->sicon), "Doing nothing...");
+		gtk_status_icon_set_tooltip_text (GTK_STATUS_ICON (watch->priv->sicon), "Doing nothing...");
 		return TRUE;
 	}
 	status = g_string_new ("");
@@ -143,7 +143,7 @@ gpk_watch_refresh_tooltip (GpkWatch *wat
 	else
 		g_string_set_size (status, status->len-1);
 
-	gtk_status_icon_set_tooltip (GTK_STATUS_ICON (watch->priv->sicon), status->str);
+	gtk_status_icon_set_tooltip_text (GTK_STATUS_ICON (watch->priv->sicon), status->str);
 	g_string_free (status, TRUE);
 	return TRUE;
 }
@@ -321,7 +321,7 @@ gpk_watch_finished_cb (PkTaskList *tlist
 		    restart == PK_RESTART_ENUM_SESSION) {
 			restart_message = gpk_restart_enum_to_localised_text (restart);
 			icon_name = gpk_restart_enum_to_icon_name (restart);
-			gtk_status_icon_set_tooltip (GTK_STATUS_ICON (watch->priv->sicon_restart), restart_message);
+			gtk_status_icon_set_tooltip_text (GTK_STATUS_ICON (watch->priv->sicon_restart), restart_message);
 			gpk_smart_icon_set_icon_name (watch->priv->sicon_restart, icon_name);
 		}
 	}
@@ -372,7 +372,7 @@ gpk_watch_finished_cb (PkTaskList *tlist
 		goto out;
 
 	/* TRANSLATORS: title: an action has finished, and we are showing the libnotify bubble */
-	notification = notify_notification_new (_("Task completed"), message, "help-browser", NULL);
+	notification = notify_notification_new (_("Task completed"), message, "help-browser");
 	notify_notification_set_timeout (notification, 5000);
 	notify_notification_set_urgency (notification, NOTIFY_URGENCY_LOW);
 	notify_notification_add_action (notification, "do-not-show-notify-complete",
@@ -443,7 +443,7 @@ gpk_watch_error_code_cb (PkTaskList *tli
 	title_prefix = g_strdup_printf ("%s: %s", _("Package Manager"), title);
 
 	/* do the bubble */
-	notification = notify_notification_new (title_prefix, message, "help-browser", NULL);
+	notification = notify_notification_new (title_prefix, message, "help-browser");
 	notify_notification_set_timeout (notification, 15000);
 	notify_notification_set_urgency (notification, NOTIFY_URGENCY_LOW);
 	notify_notification_add_action (notification, "show-error-details",
@@ -491,7 +491,7 @@ gpk_watch_message_cb (PkTaskList *tlist,
 	escaped_details = g_markup_escape_text (details, -1);
 
 	/* do the bubble */
-	notification = notify_notification_new (title_prefix, escaped_details, "help-browser", NULL);
+	notification = notify_notification_new (title_prefix, escaped_details, "help-browser");
 	notify_notification_set_timeout (notification, NOTIFY_EXPIRES_NEVER);
 	notify_notification_set_urgency (notification, NOTIFY_URGENCY_LOW);
 	ret = notify_notification_show (notification, &error);
